spec/unit/recipes/default_spec.rb

```diff
@@ -42,95 +49,42 @@
-        it do
-          expect(chef_run).to create_directory('/opt/boulder')
-        end
-        it do
-          expect(chef_run).to checkout_git('/opt/boulder')
-            .with(
-              repository: 'https://github.com/letsencrypt/boulder',
-              revision: 'release-2018-02-13'
-            )
-        end
-        it do
-          expect(chef_run).to create_docker_service('default').with(misc_opts: '--live-restore')
-        end
-        %w(boulder_config boulder_limit boulder_dns wait_for_bootstrap).each do |rb|
-          it do
-            expect(chef_run).to run_ruby_block(rb)
-          end
-        end
-        it do
-          expect(chef_run).to run_execute('/usr/local/bin/docker-compose up -d')
-            .with(
-              live_stream: true,
-              cwd: '/opt/boulder'
-            )
-        end
-      when CENTOS_6
-        %w(boulder boulder-rabbitmq boulder-mysql).each do |host|
-          it do
-            expect(chef_run).to render_file('/etc/dnsmasq.conf').with_content(%r{^address=/#{host}/127.0.0.1$})
-          end
-        end
-        it do
-          expect(chef_run).to create_directory('/opt/go/src/github.com/letsencrypt/boulder')
-        end
-        it do
-          expect(chef_run).to checkout_git('/opt/go/src/github.com/letsencrypt/boulder')
-            .with(
-              repository: 'https://github.com/letsencrypt/boulder',
-              revision: '2d33a9900cafe82993744fe73bd341fe47df2171'
-            )
-        end
-        it do
-          expect(chef_run).to install_package(%w(screen initscripts logrotate tar wget libtool-ltdl-devel))
-        end
-        it do
-          expect(chef_run).to install_python_runtime('2.7')
-        end
+      %w(boulder boulder-rabbitmq boulder-mysql).each do |host|
         it do
-          expect(chef_run).to create_cookbook_file('/opt/go/src/github.com/letsencrypt/boulder/test/setup.sh')
-            .with(mode: 0755)
-        end
-        %w(boulder_config boulder_limit wait_for_bootstrap).each do |rb|
-          it do
-            expect(chef_run).to run_ruby_block(rb)
-          end
-        end
-        it do
-          expect(chef_run).to run_bash('boulder_setup')
-            .with(
-              live_stream: true,
-              cwd: '/opt/go/src/github.com/letsencrypt/boulder',
-              code: 'source /etc/profile.d/golang.sh && GO15VENDOREXPERIMENT=1 ' \
-										'./test/setup.sh 2>&1 && touch setup.done',
-              creates: '/opt/go/src/github.com/letsencrypt/boulder/setup.done'
-            )
+          expect(chef_run).to_not render_file('/etc/dnsmasq.conf').with_content(%r{^address=/#{host}/127.0.0.1$})
         end
+      end
+      it do
+        expect(chef_run).to create_directory('/opt/boulder')
+      end
+      it do
+        expect(chef_run).to checkout_git('/opt/boulder')
+          .with(
+            repository: 'https://github.com/letsencrypt/boulder',
+            revision: 'release-2018-02-13'
```

Comment:
Note to self: using a newer version here might make the boulder verifier run nicer. This can be done in a separate PR since this is out of the scope of this PR.